desktop/core/src/desktop/conf.py

```diff
@@ -642,13 +642,16 @@ def default_from_email():
     _default_from_email = "hue@" + fqdn
   return _default_from_email
 
+from indexer.conf import ENABLE_DIRECT_UPLOAD  # Circular dependency 
 
 def default_database_options():
   """Database type dependent options"""
   if DATABASE.ENGINE.get().endswith('oracle'):
     return {'threaded': True}
   elif DATABASE.ENGINE.get().endswith('sqlite3'):
     return {'timeout': 30}
+  elif DATABASE.ENGINE.get().endswith('mysql') and ENABLE_DIRECT_UPLOAD:
+    return {'init_command': 'SET GLOBAL max_allowed_packet=67108864'}
```

Comment:
Nit: Can we add a comment for what this number denotes?

Reply:
Done